config/detekt/baseline.xml

```diff
@@ -196,6 +196,7 @@
     <ID>MagicNumber:ExpandableItemViewHolder.kt$ExpandableItemViewHolder$180F</ID>
     <ID>MagicNumber:ExpandableItemViewHolder.kt$ExpandableItemViewHolder$200</ID>
     <ID>MagicNumber:FeatureAnnouncementViewModel.kt$FeatureAnnouncementViewModel$1000</ID>
+    <ID>MagicNumber:FollowerTypesUseCase.kt$FollowerTypesUseCase$100</ID>
```

Comment:
Why this need to be committed or should it be added to .gitignore

Reply:
I think it needs to be committed. Otherwise, detekt would fail on CI and other developers' computers.

Reply:
Probably not, see this post paqN3M-yI-p2

Reply:
It's a good post, thanks! Removed added line. e03710a8a10396cf272833c664531f4fc7eea225